src/canvas.rs

```diff
@@ -554,7 +563,7 @@ impl Painter {
                 let middle_chunks = Layout::default()
                     .direction(Direction::Horizontal)
                     .constraints([Constraint::Percentage(50), Constraint::Percentage(50)])
-                    .split(vertical_chunks[2]);
+                    .split(vertical_chunks[1]);
                 self.draw_basic_cpu(&mut f, app_state, vertical_chunks[0], 1);
                 self.draw_basic_memory(&mut f, app_state, middle_chunks[0], 2);
                 self.draw_basic_network(&mut f, app_state, middle_chunks[1], 3);
```

Comment:
Ideally this isn't hard-coded in the future.  Use that layout system you spent so long on, damn it!